content/docs/started/getting-started-gke.md

```diff
@@ -168,47 +172,60 @@ Before installing Kubeflow on the command line:
+    [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/)
   * If you're using
     [Cloud Shell](https://cloud.google.com/shell/), enable 
     [boost mode](https://cloud.google.com/shell/docs/features#boost_mode).
 
 Follow these steps to deploy Kubeflow:
 
-1. Create environment variables from the OAuth client ID and secret that you
-  obtained earlier:
+1. **IMPORTANT** Create user credential: `gcloud auth application-default login`
+   * You only need to run this command once.
 
-    ```
+1. Create environment variables for your access control services:
+
+    ```bash
+    # If using IAP, create environment variables from the OAuth client ID and
+    # secret that you obtained earlier:
     export CLIENT_ID=<CLIENT_ID from OAuth page>
     export CLIENT_SECRET=<CLIENT_SECRET from OAuth page>
+
+    # If using basic auth, create environment variables for username/password:
+    export KUBEFLOW_USERNAME=<your username>
+    export KUBEFLOW_PASSWORD=<your password>
     ```
 
-1. Run the following commands to download `kfctl.sh`:
+1. Build `kfctl` binary:
 
-    ```
+    ```bash
     export KUBEFLOW_SRC=<FULL PATH TO YOUR CHOICE OF DOWNLOAD DIRECTORY>
-    mkdir ${KUBEFLOW_SRC}
-    cd ${KUBEFLOW_SRC}
-    export KUBEFLOW_TAG={{% kf-stable-tag %}}
-    curl https://raw.githubusercontent.com/kubeflow/kubeflow/${KUBEFLOW_TAG}/scripts/download.sh | bash
-     ```
+    git clone https://github.com/kubeflow/kubeflow.git  ${KUBEFLOW_SRC} \
+    --single-branch --branch v0.5-branch
+    cd ${KUBEFLOW_SRC}/bootstrap
+    make build-kfctl
+    ```
+   * Please take a look at [bootstrap guide](https://github.com/kubeflow/kubeflow/blob/master/bootstrap/developer_guide.md#building-kfctl) 
+     for more information.
    * **KUBEFLOW_SRC** - A directory where you want to download the source to.
      This value must include the full path to the directory.
-   * **KUBEFLOW_TAG** - A tag corresponding to the Kubeflow version to check 
-     out, such as `master` for the latest code.
-   * **Note:** You can also just clone the 
-     [Kubeflow repository](https://github.com/kubeflow/kubeflow) using `git`.
 
 1. Run the following scripts to set up and deploy Kubeflow:
 
-    ```
+    ```bash
+    # This is optional - to make kfctl bin easier to use.
+    export PATH=$PATH:${KUBEFLOW_SRC}/bootstrap/bin
+
     export KFAPP=<YOUR CHOICE OF APPLICATION DIRECTORY NAME>
-    ${KUBEFLOW_SRC}/scripts/kfctl.sh init ${KFAPP} --platform gcp --project ${PROJECT}
+    # Default will be using IAP.
+    kfctl init ${KFAPP} --platform gcp --project ${PROJECT}
+    # or if you want to use basic auth:
+    kfctl init ${KFAPP} --platform gcp --project ${PROJECT} --use_basic_auth
+
     cd ${KFAPP}
-    ${KUBEFLOW_SRC}/scripts/kfctl.sh generate platform
-    ${KUBEFLOW_SRC}/scripts/kfctl.sh apply platform
-    ${KUBEFLOW_SRC}/scripts/kfctl.sh generate k8s
-    ${KUBEFLOW_SRC}/scripts/kfctl.sh apply k8s
+    kfctl generate all
```

Comment:
Lets recommend using verbose log mode for now
add -V

Reply:
done.